src/main/java/edu/ucsb/cs56/mapache_search/search/SearchResult.java

```diff
@@ -31,9 +32,19 @@ public String getKind() {
         return this.items;
     }
 
+    public SearchInformation getSearchInformation() {
+        return this.searchInformation;
+    }
+
```

Comment:
Add a note about why you're max-ing the page count to 10, with a link to google api docs.

Reply:
@scottpchow23 is adding that to issue #94